neuralop/layers/integral_transform.py

```diff
@@ -138,21 +134,37 @@ def forward(
             x = y
 
         rep_features = y[neighbors["neighbors_index"]]
+
+        # batching only matters if latent embedding values are provided
```

Comment:
It is unclear what a latent embedding is in this general function. Perhaps `f_y`?